doc/guides/06-conventions.md

```diff
@@ -0,0 +1,140 @@
+# Conventions
+Interactions follow these conventions:
+
+* **Code location**: Interactions are stored in app/interactions.
```

Comment:
Can you make all these bolded words sections? Like


### Code location
Interactions are stored in app/interactions.

### Naming
Interaction names always start with a verb, optionally followed by an Object noun. The Interaction’s parent namespaces provide additional context. When referring to ActiveRecord models in an interaction’s parent namespace, use plural form. This is to avoid naming conflicts with ActiveRecord models. Example:

* `users/create.rb`